src/firefly/html/css/global.css

```diff
@@ -311,6 +311,25 @@
 }
 /*------------------- loading mask >---------------------*/
 
+.lost-connection {
+    position: fixed;
+    z-index: 200;
```

Comment:
z-index should be 300, our dialogs are at 200.  The drop downs will appears on top of the down message.